hackathon_site/dashboard/frontend/src/components/inventory/Item/Item.tsx

```diff
@@ -15,11 +15,12 @@ interface ItemProps {
 
 const Item = ({ image, title, total, currentStock }: ItemProps): ReactElement => {
     const userType = useSelector(userTypeSelector);
-    const stock = !currentStock
-        ? "OUT OF STOCK"
-        : userType === "participant"
-        ? currentStock + " IN STOCK"
-        : currentStock + " OF " + total + " IN STOCK";
+    const stock =
+        !currentStock || currentStock < 0
```

Comment:
```suggestion
currentStock < 1
```

Reply:
I think the `!currentStock` existed because we didn't have typing before and the value could have been null?